src/aws-config.js

```diff
@@ -0,0 +1,26 @@
+// WARNING: DO NOT EDIT. This file is automatically generated by AWS Amplify. It will be overwritten.
+
+const awsmobile = {
+    "aws_project_region": "us-west-2",
+    "aws_cognito_identity_pool_id": process.env.REACT_APP_AWS_COGNITO_POOL_ID,
+    "aws_cognito_region": "us-west-2",
+    "aws_user_pools_id": process.env.REACT_APP_AWS_USER_POOLS_ID,
+    "aws_user_pools_web_client_id": process.env.REACT_APP_AWS_USER_POOLS_WEB_CLIENT_ID,
+    "oauth": {
+        "domain": process.env.REACT_APP_OATH_DOMAIN,
+        "scope": [
+            "phone",
+            "email",
+            "openid",
+            "profile",
+            "aws.cognito.signin.user.admin"
+        ],
+        "redirectSignIn": "http://localhost:3000/login-redirect/",
+        "redirectSignOut": "http://localhost:3000/",
```

Comment:
will these be changed later on for production?

Reply:
Yup